Looks like this broke some stuff:

    $ dev-pipeline build
    Error: name '_EXECUTOR_TYPES' is not defined
    Traceback (most recent call last):
      File "/home/stephen/.local/bin/dev-pipeline", line 11, in <module>
        load_entry_point('dev-pipeline', 'console_scripts', 'dev-pipeline')()
      File "/home/stephen/programming/dev-pipeline/lib/devpipeline/exec/driver.py", line 45, in main
        tool(sys.argv[2:])
      File "/home/stephen/programming/dev-pipeline/lib/devpipeline/exec/build.py", line 13, in main
        devpipeline.common.execute_tool(builder, args)
      File "/home/stephen/programming/dev-pipeline/lib/devpipeline/common.py", line 161, in execute_tool
        tool.execute(args)
      File "/home/stephen/programming/dev-pipeline/lib/devpipeline/common.py", line 124, in execute
        helper_fn = _EXECUTOR_TYPES.get(parsed_args.executor)
    NameError: name '_EXECUTOR_TYPES' is not defined

--- a/lib/devpipeline/common.py
+++ b/lib/devpipeline/common.py
@@ -121,7 +121,7 @@ class TargetTool(GenericTool):
             self.targets = self.components.sections()
         self.setup(parsed_args)
         if self.verbosity:
-            helper_fn = _EXECUTOR_TYPES.get(parsed_args.executor)
+            helper_fn = EXECUTOR_TYPES.get(parsed_args.executor)
             if not helper_fn:
                 raise Exception(
                     "{} isn't a valid executor".format(parsed_args.executor))

--- a/lib/devpipeline/common.py
+++ b/lib/devpipeline/common.py
@@ -121,7 +121,7 @@ class TargetTool(GenericTool):
             self.targets = self.components.sections()
         self.setup(parsed_args)
         if self.verbosity:
-            helper_fn = EXECUTOR_TYPES.get(parsed_args.executor)
+            helper_fn = devpipeline.EXECUTOR_TYPES.get(parsed_args.executor)
             if not helper_fn:
                 raise Exception(
                     "{} isn't a valid executor".format(parsed_args.executor))
@@ -143,7 +143,7 @@ class TargetTool(GenericTool):
             self.executor.message("  {}".format(target))
             self.executor.message("-" * (4 + len(target)))
             current = self.components[target]
-            env = _create_target_environment(current)
+            env = create_target_environment(current)
 
             config_info["current_target"] = target
             config_info["current_config"] = current

